src/overlays/actors/ovl_En_Takaraya/z_en_takaraya.c

```diff
@@ -28,61 +47,360 @@ ActorInit En_Takaraya_InitVars = {
     (ActorFunc)EnTakaraya_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80ADFB20[] = {
+void* D_80ADFB00[4] = { 0x06006F58, 0x06007358, 0x06007758, 0x06007358 };
+
+void* D_80ADFB10[4] = { 0x06007B58, 0x06007F58, 0x06007758, 0x06007F58 };
+
+static InitChainEntry sInitChain[] = {
     ICHAIN_U8(targetMode, TARGET_MODE_6, ICHAIN_CONTINUE),
     ICHAIN_F32(targetArrowOffset, 1000, ICHAIN_STOP),
 };
 
-#endif
+u32 D_80ADFB28 = 0;
+
+u16 D_80ADFB2C[6] = { 1901, 1902, 1903, 1900, 1901, 0 };
+
+u8 D_80ADFB38[][2] = { { 5, 5 }, { 12, 5 }, { 4, 4 }, { 42, 41 }, { 5, 5 }, { 0, 0 } };
+
+u16 D_80ADFB44[] = { 1909, 1910, 1911, 1908, 1909, 0 };
```

Comment:
```suggestion
u16 D_80ADFB44[PLAYER_FORM_MAX] = { 
1909, // PLAYER_FORM_FIERCE_DEITY
1910, 1911, 1908, 1909, };
```
Same here, the 0 at the end is just padding. See if you can apply this logic to `D_80ADFB38` and `D_80ADFB2C` to figure out what the padding is there